.eslintrc.js

```diff
@@ -45,7 +45,6 @@ module.exports = {
 
     // TODO reactivate all the following rules
 
-    'func-names': 'off',
```

Comment:
Not really. Otherwise one of the commit would have a broken build. But I don't see how it would change anything for cherry-pick and rebase? You would have the same conflics anyway right?